web/src/frontend/src/nav-soknad/components/vedlegg/Vedlegg.tsx

```diff
@@ -27,8 +52,33 @@ class Vedlegg extends React.Component<Props, {}> {
 					htmlType="submit"
 					spinner={false}
 					disabled={false}
-					onClick={ () => {this.handleClick(); }}
+					onClick={() => {
+						this.refs.leggTilVedleggKnapp.click();
+					}}
 				>+ <FormattedMessage id="opplysninger.vedlegg.knapp.tekst"/></Knapp>
+
+				<input
+					ref="leggTilVedleggKnapp"
```

Comment:
Det er vel bedre å sette komponenten direkte gjennom ref={ (c) => this.leggTilKnapp = c }. Da slipper du å definere refs = {}
Kan sikkert også sette leggTilKnapp til typen HTMLInputElement